content/various/BumpAllocator.h

```diff
@@ -10,7 +10,7 @@
 #pragma once
 
 // Either globally or in a single class:
-static char buf[450 << 20];
+static char buf[450 << 21];
```

Comment:
```suggestion
static char buf[900 << 20];
```
Más legible (900MiB)